Types/Entities/World.js

```diff
@@ -340,7 +350,8 @@ function World(spec) {
     running = false
     setCompletionTime(null)
 
-    ui.mathField.blur()
+    ui.mathField.focus()
```

Comment:
Is this an okay change to make? We need to give the `mathField` focus when the game isn't running or the virtual keyboard's events won't have any target

Reply:
Yes I think this might have been a hack to work around some weirdness with mathquill. It certainly seems like moving the focus to the math field is what you would want when the solution stops running 🤔